packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Neo4jEnv.ps1

```diff
@@ -19,55 +19,50 @@
 
 <#
 .SYNOPSIS
-Confirms a file path is a valid Neo4j installation
+Retrieves an environment variable value
 
 .DESCRIPTION
-Confirms a file path is a valid Neo4j installation
+Retrieves an environment variable value.  This is a helper function which aids testing and mocking
 
-.PARAMETER Neo4jHome
-Full path to confirm
+.PARAMETER Name
+Name of the environment vairable
 
 .EXAMPLE
-'C:\Neo4j\neo4j-community' | Confirm-Neo4jHome 
+Get-Neo4jEnv 'Neo4jHome'
 
-Confirm the path 'C:\Neo4j\neo4j-community' is a valid Neo4j installation
-
-.EXAMPLE
-Confirm-Neo4jHome 'C:\Neo4j\neo4j-community'
-
-Confirm the path 'C:\Neo4j\neo4j-community' is a valid Neo4j installation
+Retrieves the Neo4jHome environment variable
 
 .OUTPUTS
-System.Boolean
+System.String
+Value of the environment variable
+
+Null
+Variable doesn't exist
 
 .NOTES
 This function is private to the powershell module
 
 #>
-Function Confirm-Neo4jHome 
+Function Get-Neo4jEnv
```

Comment:
The function was created to make it easy to do unit testing/mocking and to add in non-case sensitive support.  AFIAK, Windows env-vars are not case sensitive, but our code can accidentally introduce this.  This function guards against it.